nodepool/scripts/prepare_node.sh

```diff
@@ -108,12 +112,7 @@ wget https://git.openstack.org/cgit/openstack-infra/system-config/plain/install_
 # services e.g. "restart ssh". See Ubuntu bug# "bug/1492691" for details.
 # This workaround is to cheat the package upgrade to be under chroot env so
 # that it won't restart upstart/init.
-sudo cp -p /usr/bin/ischroot /usr/bin/ischroot.bak
-sudo sh -c 'echo "exit 0" > /usr/bin/ischroot'
```

Comment:
Why do we not need this any more?  If this is removed, the comment should also be deleted.